client/engine/engine.go

```diff
@@ -385,7 +407,10 @@ func (e *Engine) handleAPIEvent(apiEvent APIEvent) (EngineEvent, error) {
 
 		case virtualdefund.ObjectiveRequest:
 			e.metrics.RecordObjectiveStarted(request.Id(*e.store.GetAddress()))
-			vdfo, err := virtualdefund.NewObjective(request, true, *e.store.GetAddress(), e.store.GetChannelById, e.store.GetConsensusChannel)
+			getVoucher := func(cId types.Destination) (payments.Voucher, error) {
```

Comment:
Maybe it's simpler to just pass in the `Voucher` directly instead of a `getter` function.

Reply:
Addressed in [d5f02ae](https://github.com/statechannels/go-nitro/pull/826/commits/d5f02ae41aec53db1b424869e26b90e74621412c)